src/components/AppBar/AppBar.svelte

```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+	import type { Snippet } from "svelte";
+
+	interface AppBarProps {
+		/* Styling */
+		rootBackground?: string;
+		rootBorder?: string;
+		rootPadding?: string;
+		rootShadow?: string;
+		rootSpacing?: string;
+		rootGridColumns?: string;
+		rootGap?: string;
+		regionRowMain?: string;
+		regionRowHeadline?: string;
+		leadSnippetClasses?: string;
+		defaultSnippetClasses?: string;
+		trailSnippetClasses?: string;
+
+		/* Snippets */
+		regionRowHeadlineSnippet?: Snippet;
+		leadSnippet?: Snippet;
+		children?: Snippet;
+		trailSnippet?: Snippet;
+
+		/* Accessibility */
+		label?: string;
```

Comment:
Keep generic props (read: most non-style props) at the top of the prop definitions.

Reply:
Resolved